ERA/ERA_Fields_New.py

```diff
@@ -1157,7 +1157,7 @@ def create_mask_xarray(model:str, area:str, lsm:xr.DataArray) -> xr.DataArray:
     if model in ['ERA5', 'CESM']:
         if area == 'France':
             mask = standardize_dim_names(lsm > 0.5) # convert to bool keeping only the land masses
-            mask *= (mask.lat < 52)*(mask.lat > 42)*(mask.lon > -5)*(mask.lon < 8.3) # identify the rough region
+            mask *= (mask.lat < 52)*(mask.lat > 42)*(mask.lon % 360 > -5 % 360)*(mask.lon % 360 < 8.3 % 360) # identify the rough region
```

Comment:
any reason why we need to check this criterion?

Reply:
This way of doing it didn't work. But the point is that when you say `lon > -5` you are assuming that lon is in [-180,180], and if it is not (e.g. in [0,360]) lon > -5 will be the same as `lon > 0`. Not good.

Anyways, this is properly fixed in the new version